### PR TITLE
Remove std::copy usage.

### PR DESCRIPTION
**Pull request by Ozomahtli**
Fixes missing `<algorithm>` include error.

### CODE REVIEW

File: include/fmt/format.h [hunk 1]
```diff
@@ -2896,7 +2896,7 @@ class bigint {
     auto size = other.bigits_.size();
     bigits_.resize(size);
     auto data = other.bigits_.data();
-    std::copy(data, data + size, bigits_.data());
+    std::uninitialized_copy(data, data + size, bigits_.data());
```
Comment (vitaut) [inline include/fmt/format.h:2899]: Looks like `uninitialized_copy` is not `constexpr`. Let's replace it with

```c++
copy_str<bigit>(data, data + size, bigits_.data());
```

**Review by vitaut — CHANGES_REQUESTED** [review]
Thanks for the PR!

File: include/fmt/format.h (hunk 1 above)
Comment (Ozomahtli) [reply include/fmt/format.h]: Done!

**Review by vitaut — APPROVED** [review]

**Comment by vitaut** [comment]
Thanks!

**Pull request merged**